test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,6 +34,18 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+        // All objects are null
+        assertAnyNull(null, null);
+        assertAnyNull(null, null, null);
+
+        // Some objects are null
+        assertAnyNull("123", null);
+        assertAnyNull(null, "abc");
+
+        // All objects are not null
+        assertNotNull("123", "ab");
+        assertNotNull("123", 123, 45, "Ω");
```

Comment:
Please use a clean PR for LO submissions as it would greatly improve the readability of your changes.